### PR TITLE
Fx show hide password

### PR DESCRIPTION
**Pull request by Bukirwa99**
# Description
The hide /show password feature enables the user to either make their password visible or hidden when logging in or registering an account.

## Type of change
- [x] New feature (non-breaking change which adds functionality

## Trello Ticket ID

https://trello.com/c/9SZQYSnt

## How Can This Been Tested?
Run the branch locally and check the login and register account pages

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

# Screenshots
Screenshot 2023-03-19 182222
Screenshot 2023-03-19 182252

### CODE REVIEW

**Review by mosesmulumba — APPROVED** [review]
Thank's for the work......

File: src/pages/AdminLoginPage/AdminLoginPage.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  top: 50%;
+  right: 1rem;
+  transform: translateY(-50%);
+  cursor: pointer;
+}
+x .password-input-container {
+  position: relative;
+}
+
+.CopyIcon {
```
Comment (neelxie) [inline src/pages/AdminLoginPage/AdminLoginPage.css:12]: What does this do? You notice you copied code without understanding what it even does?

File: src/pages/AdminLoginPage/index.js [hunk 2]
```diff
@@ -99,16 +112,26 @@ class AdminLoginPage extends React.Component {
...
+                  placeholder="Password"
+                  name="password"
+                  type={passwordShown ? "text" : "password"}
+                  value={password}
+                  onChange={(e) => {
+                    this.handleChange(e);
+                  }}
+                />
+                <div className="CopyIcon">
+                  <CopyText onClick={this.passwordOnClick} />
```
Comment (neelxie) [inline src/pages/AdminLoginPage/index.js:127]: What does this do? how is it even necessary, goes to show you copied code without having an understanding of how it works, please try to understand what is required and then implement

File: src/pages/LoginPage/LoginPage.css [hunk 3]
```diff
@@ -116,3 +114,34 @@
     width: 100%;
   }
 }
+.password-wrapper {
+  display: flex;
+  align-items: center;
+  height: 2.5rem;
```
Comment (neelxie) [inline src/pages/LoginPage/LoginPage.css:120]: remove this as it makes the password input smaller compared to the other inputs.

**Review by neelxie — CHANGES_REQUESTED** [review]
You are duplicating alot of code and clearly you copied code, you didn't understand, please revisit your code and remove unnecessary code and ofcourse follow the design.

**Review by rhodinemma — APPROVED** [review]
Thank you for this. @Bukirwa99 
But your PR has conflicts, check them out ASAP.

**Review by neelxie — APPROVED** [review]
LGTM

**Pull request merged**